Xext/xres.c

```diff
@@ -396,57 +377,43 @@ static Bool
 ConstructClientIdValue(ClientPtr sendClient, ClientPtr client, CARD32 mask,
                        ConstructClientIdCtx *ctx)
 {
-    xXResClientIdValue reply = {
-        .spec.client = client->clientAsMask,
-    };
-
-    if (client->swapped) {
-        swapl (&reply.spec.client);
-    }
-
     if (WillConstructMask(client, mask, ctx, X_XResClientXIDMask)) {
-        void *ptr = AddFragment(&ctx->response, sizeof(reply));
-        if (!ptr) {
-            return FALSE;
-        }
+        xXResClientIdValue reply = {
+            .spec.client = client->clientAsMask,
+            .spec.mask = X_XResClientXIDMask
+        };
 
-        reply.spec.mask = X_XResClientXIDMask;
-        if (sendClient->swapped) {
+        if (client->swapped) {
```

Comment:
In the old code it’s `sendClient`, in the new it’s `client`. Which one is correct?